### DISCUSSION TITLE
Update failed without reasons (correlationId)

### DISCUSSION THREAD

**Discussion by corentin-begne** [microsoft/MicrosoftEdge-Extensions#120] [Discussions]
Hi,
I was able to submit my first extension version but when I tri to update it i've an error about something went wrong and try again later with correlationId : uuid but no reasons or reports.
My update works fine on chrome and firefox and was validated on upload, I tried it in Edge and it works without any errors.
How can I know why it's working ?

**Comment by Bhuvana-Priya** [comment]
Hi @corentin-begne,

Thank you for reporting this. 

Could you please provide more details about your extension @ext_dev_support@microsoft.com

Regards,
Bhuvana

**Comment by corentin-begne** [comment]
Hi,
Ty for you reply.
I found the problem, it was because my javascript files had the mjs extension ... it seems that it's mandatory to set .js even for modules but it worked at first like this and the error message doesn't explain anything.
